packages/modules/src/modules/banners/choiceCardsBanner/components/ChoiceCards.tsx

```diff
@@ -149,7 +149,7 @@ export const ChoiceCards: React.FC<ChoiceCardProps> = ({
                 submitComponentEvent({
                     component: {
                         componentType: 'ACQUISITIONS_OTHER',
-                        id: componentId,
+                        id: 'contributions-banner-choice-cards',
```

Comment:
D&I need to track impressions of the amounts test, so this needs to be consistent